tests/eval_files/async912.py

```diff
@@ -0,0 +1,108 @@
+# ASYNCIO_NO_ERROR
+import trio
+
+
+async def foo():
+    with trio.move_on_after(0.1):  # error: 4
+        ...
+    with trio.move_on_at(0.1):  # error: 4
+        ...
+    with trio.fail_after(0.1):  # error: 4
+        ...
+    with trio.fail_at(0.1):  # error: 4
+        ...
+    with trio.CancelScope(0.1):  # error: 4
+        ...
+
+    with open(""):
+        ...
```

Comment:
add a comment to explain?  ('check that non-cancel-scope context managers are unaffected', or something)